### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by AmineDerbal**
in this milestone ♻️ 

- delete all the boilerplate unused files 👍 
- create a components directory inside the src folder 👍 
- create a calculator component that matches this [template](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/calculator.png?raw=true) 👍 
- call calculator from App.js file 👍 
- calculator is rendered when the page load 👍

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        ÷
+      </button>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="orange-button">
+        x
+      </button>
+      <button type="button">4</button>
+      <button type="button">5</button>
```
Comment (DuaneDave) [inline src/components/Calculator.js]: According to the requirement, there should be a child component within this Calculator component.

for example:

```
Function Calculator() {
  return (
    <SomeComponents />
    // YOUR JSX GOES HERE
  )
}
```

Consider doing this in other to stick to the project requirements.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @AmineDerbal 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## Highlights
✔️ Your design looks good
✔️ Linters are passing
✔️ Correct use of gitflow

## Required Changes ♻️
- It is stated in the requirement to remove all the files that comes with the CRA boilerplate. I have marked the files to be removed from the set up.

bad


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Structuring your readme to match the table of content in order to make it more organized.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmineDerbal** [comment]
- Remove all boilerplate files 👍 
- create component Screen 👍 
- use Screen as a child Component in calculator 👍

**Review by DammyShittu — APPROVED** [review]
Hi @AmineDerbal,

## APPROVED 🟢

## Highlights
✔️ Child component in Calculator

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**